server/blob/blob-objectstorage/src/main/java/org/apache/james/blob/objectstorage/ObjectStorageBlobsDAO.java

```diff
@@ -119,20 +121,24 @@
 
     @Override
     public InputStream read(BlobId blobId) throws ObjectStoreException {
-        Blob blob = blobStore.getBlob(containerName.value(), blobId.asString());
+        Optional<Blob> retrievedBlob = Mono.fromCallable(() -> Optional.ofNullable(blobStore.getBlob(containerName.value(), blobId.asString())))
+            .retryBackoff(5, Duration.ofMillis(100))
```

Comment:
magic numbers ?

Reply:
Do you want some configuration or Constants are enough?

Reply:
constants are ok.